### PR TITLE
Selector and query factory optimisations

### PR DESCRIPTION
**Pull request by frup42**

### CODE REVIEW

File: fflib/src/classes/fflib_SObjectSelector.cls [hunk 1]
```diff
@@ -107,9 +113,20 @@ public abstract with sharing class fflib_SObjectSelector
      **/
     public fflib_SObjectSelector(Boolean includeFieldSetFields, Boolean enforceCRUD, Boolean enforceFLS)
     {
+        this(includeFieldSetFields, enforceCRUD, enforceFLS, true);
+    }
+
+    /**
+     * Constructs the Selector
+     *
+     * @param includeFieldSetFields Set to true if the Selector queries are to include Fieldset fields as well
```
Comment (afawcett) [inline fflib/src/classes/fflib_SObjectSelector.cls]: Should this doc section describe the new sort parameter? @frup42

Comment (frup42) [inline fflib/src/classes/fflib_SObjectSelector.cls]: Yes, I've added comments for all 4 params.

**Comment by phyopine-gscs** [comment]
@frup42 , Thank you. I was looking at the same things over the holidays as well. Glad you beat me to it. :-)

**Comment by afawcett** [comment]
Thanks @frup42  :+1:

**Comment by capeterson** [comment]
Many thanks @frup42, I was hoping to see this make it's way to github!

**Pull request merged**